lib/events.js

```diff
@@ -425,9 +425,9 @@ EventEmitter.prototype.listeners = function listeners(type) {
     if (!evlistener)
       ret = [];
     else if (typeof evlistener === 'function')
-      ret = [evlistener];
+      ret = [evlistener.listener ? evlistener.listener : evlistener];
```

Comment:
Sure thing, pushing that up. This was just mirroring the check as written for `'newListener`'.